src/lib.rs

```diff
@@ -1329,6 +1367,31 @@ impl<'ctx> Bindings<'ctx> {
         try!(eof(&mut ps.s));
         ps.s.out.flush()
     }
+
+    /// Checks if format_bindings is set and runs rustfmt on the file
+    fn format_generated_file(&self, file: &Path) {
+        if !self.context.options().format_bindings {
+            return;
+        }
+
+        let rustfmt = if let Ok(rustfmt) = which::which("rustfmt") {
+            rustfmt
+        } else {
+            error!("Could not find rustfmt in the global path.");
+            return;
+        };
+
+        let mut cmd = Command::new(rustfmt);
+
+        if let Some(path) = self.context.options().format_configuration_file.as_ref().and_then(
+            |f| f.to_str()) {
+            cmd.args(&["--config-path", path]);
+        }
+
+        if let Err(e) = cmd.arg(file).status() {
```

Comment:
Note that `status()` only returns the child processes exit status, it doesn't check that the exit status is success. Additionally, I think it makes sense to propagate any errors, since (unlike if `rustfmt` is just missing) we don't know if the bindings are still in an OK state if this fails: maybe only half of the formatted bindings got written to the file, and the other half didn't?

```
let status = cmd.arg(file).status()?;
if status.success() {
    Ok(())
} else {
    Err(io::Error::new(io::ErrorKind::Other, "rustfmt exited with a failure status: {}", status))
}
```